### PR TITLE
[FIX] addressPopover: use $templateRequest (RT-3357)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
addressPopover: load addresspopover.html instead of the jade file
using $templateRequest

### CODE REVIEW

File: src/js/directives/addressPopover.directive.js [hunk 1]
```diff
@@ -212,9 +209,16 @@
    */
   angular.module('directives').directive('rpAddressPopover', rpAddressPopover);
 
-  rpAddressPopover.$inject = ['$timeout', 'rpId', '$filter', '$parse'];
+  rpAddressPopover.$inject = ['$timeout', 'rpId', '$filter', '$parse', '$templateRequest'];
+
+  function rpAddressPopover($timeout, id, $filter, $parse, $templateRequest) {
+    $templateRequest('templates/' + window.lang + '/directives/addresspopover.html', false).then(function(template) {
+      console.log(template);
```
Comment (vhpoet) [inline src/js/directives/addressPopover.directive.js]: Please remove the console.log

**Comment by darkdarkdragon** [comment]
@vhpoet done

**Comment by vhpoet** [comment]
LGTM

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**